cinema/serializers.py

```diff
@@ -135,3 +136,43 @@
 
 class OrderListSerializer(OrderSerializer):
     tickets = TicketListSerializer(many=True, read_only=True)
+
+
+class MovieImageSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Movie
+        fields = ['id', 'image']
+
+
+class MovieSerializer(serializers.ModelSerializer):
+    image = serializers.ImageField(source='image.url', read_only=True)
+
+    class Meta:
+        model = Movie
+        fields = ['id', 'title', 'description', 'duration', 'image']
+
+
+class MovieSessionSerializer(serializers.ModelSerializer):
+    movie_image = serializers.ImageField(source='movie.image.url', read_only=True)
+
+    class Meta:
+        model = MovieSession
+        fields = ['id', 'movie', 'movie_image', 'session_time', 'available_seats']
```

Comment:
The `MovieSessionSerializer` is also defined twice. This can cause conflicts and unexpected behavior. Make sure to consolidate the definitions and ensure each serializer is uniquely defined.